README.md

```diff
@@ -338,6 +338,25 @@ afterpayBreakdown.introText = AfterpayIntroText.MAKE_TITLE
 
 Given the above, the price breakdown text will be rendered `Make 4 interest-free payments of $##.##`
 
+##### More Info Options
+Setting `moreInfoOptions` is optional and of type `AfterpayMoreInfoOptions`. This class takes three parameters:
+- `modalId`: a `string` that is the filename of a modal hosted on Afterpay static.
+- `modalTheme`: an enum of type `AfterpayModalTheme` withe the following options: `DEFAULT`, `DEFAULT_CBT`, `WHITE` and `WHITE_CBT`.
```

Comment:
Small typo.

```suggestion
- `modalTheme`: an enum of type `AfterpayModalTheme` with the following options: `DEFAULT`, `DEFAULT_CBT`, `WHITE` and `WHITE_CBT`.
```

Reply:
Also, is it possible to make this consistent with `ei-lib-js` where `data-modal-theme` supports `MINT` and `WHITE` and cbt modals are loaded based on the value (`true`/`false`) set for the `data-is-cbt` attribute? Ignore this comment if it's not possible technically.

Reply:
Thanks for these two pickups. Committed the typo but the functionality for `isCBT` is how it works per the next line in the documentation. The above line was the way it was initially implemented and just wasn't removed. So fix will be to update the docs.

Reply:
addressed in 8e71181